### PR TITLE
@alloy => Adds lot numbers

### LINKED ISSUE #362: Add lot numbers to grid view and artwork view
in grid view: avant garde 24px
http://cl.ly/image/472o3H0d3N3r
image

in list view: avant garde 24px
image

on artwork view: avant garde 24px
image

### PR DESCRIPTION
**Pull request by ashfurrow**
Lots of small but related changes. For example, removed reference image references from the Xcode project to prevent compiler errors for failing to copy images that have been renamed, as well as un-pending'd a few tests (and made them pass). 

- [x] Listing table cells
- [x] Listing grid cells
- [x] Sale artwork view controller
- [x] Bid fulfillment view controller

Fixes #362.

### CODE REVIEW

**Comment by ashfurrow** [comment]
@alloy this is good for review finally!

File: Kiosk/App/Models/SaleArtwork.swift [hunk 1]
```diff
@@ -158,6 +171,8 @@ public class SaleArtwork: JSONAble {
     override public class func keyPathsForValuesAffectingValueForKey(key: String) -> NSSet {
         if key == "estimateString" {
             return NSSet(array: ["lowEstimateCents", "highEstimateCents"])
+        } else if key == "lotNumberSignal" {
+            return NSSet(array: ["lotNumber"])
```
Comment (alloy) [inline Kiosk/App/Models/SaleArtwork.swift]: Is there a special reason why you are not using `+[NSSet setWithObject:]` ?

Comment (ashfurrow) [inline Kiosk/App/Models/SaleArtwork.swift]: No – good idea.

File: KioskTests/Bid Fulfillment/PlaceBidViewControllerTests.swift [hunk 2]
```diff
@@ -4,26 +4,64 @@ import ReactiveCocoa
...
+
+            beforeEach {
+                sut = sharedExampleContext()["sut"] as PlaceBidViewController!
+                nav = sharedExampleContext()["nav"] as FulfillmentNavigationController!
+            }
+
+            describe("with lot number") {
+                beforeEach {
+                    nav.bidDetails.saleArtwork?.lotNumber = 13
+                    return
```
Comment (alloy) [inline KioskTests/Bid Fulfillment/PlaceBidViewControllerTests.swift]: Why are you using an explicit `return` here? (More of Swift question, I assume.)

File: KioskTests/ListingsViewControllerTests.swift [hunk 3]
```diff
@@ -6,6 +6,47 @@ import Nimble_Snapshots
...
+            var sut: ListingsViewController!
+
+            beforeEach{
+                sut = sharedExampleContext()["sut"] as ListingsViewController!
+            }
+
+            it("grid") {
+                sut.switchView[0]?.sendActionsForControlEvents(.TouchUpInside)
+                expect(sut) == snapshot()
+            }
```
Comment (alloy) [inline KioskTests/ListingsViewControllerTests.swift]: Intentional whitespace missing?

File: KioskTests/Bid Fulfillment/PlaceBidViewControllerTests.swift (hunk 2 above)
Comment (ashfurrow) [inline KioskTests/Bid Fulfillment/PlaceBidViewControllerTests.swift]: Yup – a bug in Swift fixed in 1.2 (still in beta): `beforeEach` accepts a closure as a property and that closure needs to return `Void`, essentially. The compiler will often misinterpret single-line closures that are supposed to return nothing as single-line closures that are omitting the optional `return` statement and instead want to return the results of the assignment expression.

File: KioskTests/ListingsViewControllerTests.swift (hunk 3 above)
Comment (ashfurrow) [inline KioskTests/ListingsViewControllerTests.swift]: Nope, thanks!

File: KioskTests/Bid Fulfillment/PlaceBidViewControllerTests.swift (hunk 2 above)
Comment (alloy) [inline KioskTests/Bid Fulfillment/PlaceBidViewControllerTests.swift]: Fun stuff! Thanks for the info.

**Comment by ashfurrow** [comment]
@alloy good to merge?

**Comment by alloy** [comment]
@ashfurrow Looks good to me :+1:

(oops, forgot to hit merge)

**Pull request merged**